source/MaterialXRenderGlsl/GlslRenderer.h

```diff
@@ -129,6 +129,7 @@ class MX_RENDERGLSL_API GlslRenderer : public ShaderRenderer
     SimpleWindowPtr _window;
     GLContextPtr _context;
     Color4 _clearColor;
+    std::string _extension;
```

Comment:
As above, let's use the existing extension property of TextureBaker, since renderers themselves don't need to be aware of texture baking filenames.